### ISSUE TITLE
two websocket modules 

### ISSUE THREAD

**Issue by zoggy** [vbmithr/ocaml-websocket#40]
Hello,

The META file indicates that the package `websocket.lwt` requires the package `websocket`. But `websocket_lwt.cma` already embed `websocket.cmo`, resulting in this warning:
````
Warning 31: files .../.opam/4.02.1/lib/websocket/websocket_lwt.cma(Websocket) \
and .../.opam/4.02.1/lib/websocket/websocket.cma(Websocket) both define \
a module named Websocket
````
I think `websocket.cmo` should not be included in `websocket_lwt.cma` and let ocamlfind add it at lik time using the dependencies given in `META`.

There is the same problem with `websocket_lwt.cmxa` including both `websocket.cmx` and `websocket_lwt.cmx` but in this case this is an error instead of a warning.

**Comment by vbmithr** [comment]
Fixed in 782f90d79e0d417e233f3315fd3f412c3e583df6. Closing #40

**Comment by zoggy** [comment]
Thanks :)
